config/config.ts

```diff
@@ -24,6 +24,7 @@ export interface IDevOpsTeam {
   collection: string;
   project: string;
   repository: string;
+  filters?: RegExp[]; // Matches pull requests whos names match one of those regex
```

Comment:
> Matches pull requests
assumes that the filter is exclusive according to the code - so maybe changing this to 
```suggestion
  filters?: RegExp[]; // Excludes pull requests who's title matches one of the regular expressions
```

Reply:
🙏